pkgs/applications/misc/ulauncher/default.nix

```diff
@@ -63,6 +70,7 @@ python27Packages.buildPythonApplication rec  {
 
   patches = [
     ./fix-path.patch
+    ./fix-permissions.patch
```

Comment:
https://github.com/NixOS/nixpkgs/pull/81207#issuecomment-594275543 Please PR this patch and reference it here .

Reply:
https://github.com/Ulauncher/Ulauncher/pull/523